It seems that cmake directory doesn't contains FindProtobuf.cmake

When I attempt to install the latest branch using `ninja -C build/release install`, it gives me an error stating that it cannot find `cmake/FindProtobuf.cmake`. Even though I have copied the file with the same name from `/usr/local/share/cmake-3.28/Modules`, it still shows me error. Clearly, the recently introduced Protobuf dependency is preventing the master branch from being easily installed into `/usr/local`.

```
CMake Error at cmake_install.cmake:100 (file):
  file INSTALL cannot find "/root/seastar/cmake/FindProtobuf.cmake": No such
  file or directory.


FAILED: CMakeFiles/install.util 
cd /root/seastar/build/release && /usr/local/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.
```

```
CMake Error at /usr/local/lib/cmake/Seastar/FindProtobuf.cmake:483 (include):
  include could not find requested file:

    /usr/local/lib/cmake/Seastar/SelectLibraryConfigurations.cmake
Call Stack (most recent call first):
  /usr/local/lib/cmake/Seastar/SeastarDependencies.cmake:152 (find_package)
  /usr/local/lib/cmake/Seastar/SeastarConfig.cmake:33 (seastar_find_dependencies)
  CMakeLists.txt:11 (find_package)


-- Configuring incomplete, errors occurred!
CMake Error at /usr/local/lib/cmake/Seastar/FindProtobuf.cmake:508 (select_library_configurations):
  Unknown CMake command "select_library_configurations".
Call Stack (most recent call first):
  /usr/local/lib/cmake/Seastar/FindProtobuf.cmake:545 (_protobuf_find_libraries)
  /usr/local/lib/cmake/Seastar/SeastarDependencies.cmake:152 (find_package)
  /usr/local/lib/cmake/Seastar/SeastarConfig.cmake:33 (seastar_find_dependencies)
  CMakeLists.txt:11 (find_package)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1359,7 +1359,6 @@ if (Seastar_INSTALL)
     FILES
       ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindGnuTLS.cmake
       ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindLinuxMembarrier.cmake
-      ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindProtobuf.cmake
       ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindSanitizers.cmake
       ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindSourceLocation.cmake
       ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindStdAtomic.cmake

